packages/renderer/src/lib/agent-workspaces/AgentWorkspaceDetailsSettings.svelte

```
@@ -188,6 +243,14 @@ async function handleBrowseCustomPath(index: number): Promise<void> {
   }
 }
 
+function navigateToKnowledges(): void {
+  handleNavigation({ page: NavigationPage.RAG_ENVIRONMENTS });
+}
+
+function navigateToSkills(): void {
+  handleNavigation({ page: NavigationPage.SKILLS });
+}
+
 function navigateToSkills(): void {
```

Biome (2.4.15)

[error] 'navigateToSkills' is redeclared in the same scope.

(lint/suspicious/noRedeclare)